queue/group_remote_mongo.go

```diff
@@ -59,7 +62,7 @@ type MongoDBQueueGroupOptions struct {
 	TTL time.Duration
 }
 
-func (opts *MongoDBQueueGroupOptions) constructor(ctx context.Context, name string) remoteQueue {
+func (opts *MongoDBQueueGroupOptions) constructor(ctx context.Context, name string, rhOpts amboy.RetryHandlerOptions) (remoteQueue, error) {
```

Comment:
Why are passing in `RetryHandlerOptions` when it is already a field in `MongoDBQueueGroupOptions`?

Reply:
Removed it.